modules/renderer/src/svgCreator.ts

```diff
@@ -345,3 +345,21 @@ function createAndBindWithGraph(entity: Entity, name: string) {
   new SvgObject(entity, svgNode)
   return svgNode
 }
+export function svgSave(svg: SVGAElement, graphName: string) {
+  if (svg == null || graphName == null) return
+  saveSvg(svg, graphName + '.svg')
+}
+
+function saveSvg(svgEl: SVGAElement, name: string) {
```

Comment:
This should not be part of the renderer. You should expose a `getSvgString()` method then move the download part to the demo app.